linux_mac/magpi-special-issue-downloader.sh

```diff
@@ -10,54 +10,56 @@
 # VERSION=0.1.0
 # USAGE="Usage: sh magpi-special-issue-downloader.sh"
 
+BASEDIR=`dirname $0`/..
+OUTDIR=$BASEDIR/special_issues
 
-if [ ! -d "special_issues" ]; then
- mkdir special_issues
+if [ ! -d "$OUTDIR" ]; then
+ mkdir $OUTDIR
 fi
 
 # Essential Games v1
-wget https://www.raspberrypi.org/magpi-issues/Essentials_Games_v1.pdf -P special_issues/
+wget -N https://www.raspberrypi.org/magpi-issues/Essentials_Games_v1.pdf -P $OUTDIR
 
 # Project Book - v1 https://www.raspberrypi.org/magpi/issues/projects-1/
-wget https://raspberrypi.org/magpi-issues/Projects_Book_v1.pdf -P special_issues/
+wget -N https://raspberrypi.org/magpi-issues/Projects_Book_v1.pdf -P $OUTDIR
```

Comment:
oh, this is a real outdated version, you were using.
have a look, how it's at the moment... there is complete new structure....